commandapi-platforms/commandapi-bukkit/commandapi-bukkit-vh/src/main/java/dev/jorel/commandapi/CommandAPIVersionHandler.java

```diff
@@ -68,9 +70,54 @@ public interface CommandAPIVersionHandler {
 				case "1.20.3", "1.20.4" -> new NMS_1_20_R3();
 				case "1.20.5", "1.20.6" -> new NMS_1_20_R4();
 				case "1.21", "1.21.1" -> new NMS_1_21_R1();
-				default -> throw new UnsupportedVersionException(version);
+				default -> null;
```

Comment:
The `null` here and the intermediate `platform` variable could be avoided using the [`yield`](https://www.baeldung.com/java-yield-switch) keyword:

```java
return switch (version) {
	case "1.16.5" -> new NMS_1_16_R3();
	// ...
	case "1.21", "1.21.1" -> new NMS_1_21_R1();
	default -> {
		if (CommandAPI.getConfiguration().shouldBeMoreLenientForMinorVersions()) {
			yield minecraftVersion.matchesPatch((NMS<?>) latestNMS);
		}
		throw new UnsupportedVersionException(version);
	}
};
```

(_Dunno why GitHub doesn't render it red like the other keywords, it totally exists :P_)

Reply:
They refuse to yield to change